src/spawn.rs

```diff
@@ -1,166 +1,63 @@
-use std::future::Future;
-
-#[cfg(feature = "async_std")]
-pub use async_std_impl::*;
-#[cfg(feature = "smol")]
-pub use smol_impl::*;
+/// Spawns the given actor into the tokio runtime, returning an [`Address`] to it.
+///
+/// The 2nd parameter is the mailbox size with `None` creating an unbounded mailbox.
 #[cfg(feature = "tokio")]
-pub use tokio_impl::*;
-#[cfg(feature = "wasm_bindgen")]
-pub use wasm_bindgen_impl::*;
+#[cfg_attr(docsrs, doc(cfg(feature = "tokio")))]
+pub fn spawn_tokio<A>(actor: A, mailbox_size: Option<usize>) -> crate::Address<A>
```

Comment:
With #133, I'd like to make this parameter the actual mailbox and not just mailbox size so we can also fix #43 by making the constructors on `Mailbox` say `bounded` and `unbounded`.

For now, I chose to just take the size here because I figured it is an easier transition overall.

Reply:
That makes sense!